virt/virtual_machines/vm_networking/virt-hot-plugging-network-interfaces.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY                                     
+[id="virt-hot-plugging-network-interfaces"]                            
+= Hot plugging secondary network interfaces                                            
+include::_attributes/common-attributes.adoc[]                  
+:context: virt-hot-plugging-network-interfaces                        
+                                                                
+toc::[]     
+
+You can add or remove secondary network interfaces without stopping your virtual machine (VM). {VirtProductName} supports hot plugging and hot unplugging for Linux bridge interfaces that use the VirtIO device driver.
+
+:FeatureName: Hot plugging and hot unplugging bridge network interfaces
+include::snippets/technology-preview.adoc[]
+
+[id="virtio-limitations_{context}"]
+== VirtIO limitations
+Each VirtIO interface uses one of the limited Peripheral Connect Interface (PCI) slots in the VM. There are a total of 32 slots available. The PCI slots are also used by other devices and must be reserved in advance, therefore slots might not be available on demand. {VirtProductName} reserves up to four slots to be used later for hot plugging interfaces. If your VM has four or more existing interfaces, you cannot hot plug any additional network interfaces. 
```

Comment:
The suffix " E.g. given two plugged interface, OCP Virt assures 2 more interfaces can be hotplug." is wrong, Openshift virtualization preserves 4, if you have 4 it won't add more. But you may be able to hotplug (q35 preserves 1 interface).

I would rephrase it to - " Four network interfaces are guaranteed to be available for hot pluging, counting in also the existing plugged network interfaces.
 E.g. given two plugged interface, Openshift Virtualization assures 2 more interfaces can be hotplug."